packages/main/src/resolvers/conversation.test.ts

```diff
@@ -18,18 +18,23 @@ jest.mock("imap")
 let account: cache.Account
 let accountId: cache.ID
 let connectionManager: ConnectionManager
-let conversation: Conversation
-let conversationId: string
 
-describe("same sync tests", () => {
+beforeEach(async () => {
+  const { lastInsertRowid } = db
+    .prepare("insert into accounts (email) values (?)")
+    .run("jesse@sitr.us")
+  accountId = lastInsertRowid
+
+  connectionManager = mockConnection()
+})
+
+describe("when addressing conversations", () => {
+  let conversation: Conversation
+  let conversationId: string
+
   beforeEach(async () => {
-    const { lastInsertRowid } = db
-      .prepare("insert into accounts (email) values (?)")
-      .run("jesse@sitr.us")
-    accountId = lastInsertRowid
     account = { id: accountId, email: "jesse@sitr.us" }
```

Comment:
Please either move this assignment to the `beforeEach` above, or move the `account` variable declaration into this `describe` block.